### PR TITLE
docs :translate the zh to en

### LINKED ISSUE #545: translate: `docs/en/development/github-workflows.md` to English
**What would you like to be added**:

translate: `docs/en/development/github-workflows.md` to English

### PR DESCRIPTION
**Pull request by NTH19**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #545 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by NTH19** [comment]
I have already signed the CLA.

**Comment by seeflood** [comment]
> I have already signed the CLA.

Ok. 
I closed and reopened the PR to rerun the workflow

File: docs/en/development/github-workflows.md [hunk 1]
```diff
@@ -1,106 +1,115 @@
 # Layotto GitHub Workflows
 
-本文档解释了 Layotto 的 Github 四大工作流：
+This document explains Layotto's  four workflows in Github ：
```
Comment (Xunzhuo) [inline docs/en/development/github-workflows.md:3]: remove the extra space here

File: docs/en/development/github-workflows.md [hunk 2]
```diff
@@ -1,106 +1,115 @@
...
+
 + Layotto Env Pipeline 🌊
 + Layotto Dev Pipeline 🌊 (Before Merged)
 + Layotto Dev Pipeline 🌊 (After Merged)
 + Layotto Release Pipeline 🌊
 
-工作流中包含了一个或多个任务，提高了 Layotto 的代码规范度和安全性，简化了开发/构建/发布等重复步骤，以下为对上述四大工作流的详细解释。
+The workflow contains one or more tasks，It improves the standardization and security of the code in layotto，simplifies repetitive steps of development / build / release.The following is a detailed explanation of the above four workflows.
```
Comment (Xunzhuo) [inline docs/en/development/github-workflows.md:10]: Need a space here: `release.The`

File: docs/en/development/github-workflows.md [hunk 3]
```diff
@@ -109,37 +118,37 @@ Layotto Dev Pipeline (Before Merged) 流水线主要负责 Layotto 提交 PR 后
...
-+ Linux ARMD64 Image  发布最新版本的 Layotto 镜像，镜像规范为 layotto/layotto.arm64:latest
++ Go Style Check：Check the style of the code
++ Go CI Linter：Perform linter specification of verification on the code
++ Go Unit Test: Unit test the code
++ Coverage Analysis: Coverage analysis of the code
++ Integrate with WASM :WASM integration test on the code
++ Integrate with Runtime :Run time integration test on the code
++ Darwin AMD64 Artifact :Build Darwin AMD64 binary verification for code
++ Darwin ARM64 Artifact :Build Darwin arm64 binary verification for code
++ Linux AMD64 Artifact :Build linux amd64binary verification for code
```
Comment (Xunzhuo) [inline docs/en/development/github-workflows.md:133]: need a space here

File: docs/en/development/github-workflows.md [hunk 4]
```diff
@@ -109,37 +118,37 @@ Layotto Dev Pipeline (Before Merged) 流水线主要负责 Layotto 提交 PR 后
...
-        branches: [main] 合并 PR
-        paths-ignore: 忽略以下变更： docs 目录下文件，markdown 文件
+        branches: [main] merge PR
+        paths-ignore: ignore the following changes： docs directory files，markdown files
         - 'docs/**'
         - '**/*.md'
     pull_request:
-        branches: "*" 提交 PR
-        paths-ignore: 忽略以下变更： docs 目录下文件，markdown 文件
+        branches: "*" commit PR
```
Comment (Xunzhuo) [inline docs/en/development/github-workflows.md:150]: create or start a PR

File: docs/en/development/github-workflows.md [hunk 5]
```diff
@@ -1,106 +1,115 @@
...
-+ Title Validation (基于 Semantic 风格 检查 PR Title 的规范)
-+ Quickstart Validation (QuickStart 文档的校验)
-+ Update Stale Status (Issue/PR 状态的更新)
-+ License Validation (License 的校验)
-+ DeadLink Validation (文档中 DeadLink 的检查)
-+ CodeQL (CodeQL 的分析)
+#### Job Trigger Method
 
-#### Job 触发方式
+Layotto Env Pipeline Task Trigger Method：
```
Comment (Xunzhuo) [inline docs/en/development/github-workflows.md:27]: Layotto Env Pipeline Task Trigger Method -> Layotto Env Pipeline Task Trigger Events

File: docs/en/development/github-workflows.md [hunk 6]
```diff
@@ -1,106 +1,115 @@
...
 
-#### Job 任务内容
+#### Job Task Content
+
+Layotto Env Pipeline is mainly responsible for the project of layotto and the specification of relevant environment，it current contains the following tasks：
+
++ Title Validation (Check the specification of PR title based on semantic style)
++ Quickstart Validation (Verification of QuickStart documents)
++ Update Stale Status (Update of issue / PR status)
++ License Validation (Verification of license )
```
Comment (Xunzhuo) [inline docs/en/development/github-workflows.md:21]: remove the extra space here

**Review by Xunzhuo — CHANGES_REQUESTED** [review]
Some minor changes, thanks @NTH19

**Comment by NTH19** [comment]
> Some minor changes, thanks @NTH19

Is there a description file for the documentation specification ? @Xunzhuo

**Review by Xunzhuo — APPROVED** [review]

**Comment by Xunzhuo** [comment]
lgtm

**Review by seeflood — APPROVED** [review]
lgtm. Thanks!

**Pull request merged**